.github/workflows/maven.yml

```diff
@@ -23,6 +23,10 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - name: Cleanup Docker system
+      run: docker system prune -af
+    - name: Output system disk space
+      run: df -h
```

Comment:
Maybe you should also print out some CPU/mem statistics to see if RAM gets filled up

Reply:
it looks like `docker system prune -af` cleanups enough memory for the build. now the issue is in embedded-pulsar, looking into it now

Reply:
you are right. I found "The error message "Not enough non-faulty bookies available" indicates the bookies are not writable. It typically means that your disk usage is larger than 95% so the bookies are forced to run in readonly mode."